### ISSUE TITLE
Pager theming with Liquid

### ISSUE THREAD

**Issue by DannyT** [OrchardCMS/OrchardCore#2494]
From a combination of [these Navigation docs](https://orchardcore.readthedocs.io/en/dev/OrchardCore.Modules/OrchardCore.Navigation/README/#pager), and [this liquid file in the blog theme](https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Themes/TheBlogTheme/Views/Blog-ListPart.liquid) it appears that the properties detailed in the docs are settable from the liquid tag.

As demonstrated in the blog theme you can set the `previousText` and `nextText` values (also the next and previous classes). However other properties, such as `Tag` or `itemTag` do not seem to have any effect. 

Changing the Blog-ListPart.liquid template (or creating a Blog__ListItem template in the dashboard for convenience) to the following has no effect on the tags rendered:

```
{% for item in Model.ContentItems %}
    {{ item | shape_build_display: "Summary" | shape_render }}
{% endfor %}

{% assign previousText = "← Newer Posts" | t %}
{% assign nextText = "Older Posts →" | t %}
{% assign previousClass = "previous" | t %}
{% assign nextClass = "next" | t %}
{% assign myTag = "div" | t %}

{% shape_pager Model.Pager previous_text: previousText, next_text: nextText,
    previous_class: previousClass, next_class: nextClass, tag:myTag, item_tag:myTag %}

{{ Model.Pager | shape_render }}
```

Should this not work or am I misunderstanding the inter-changeability between liquid and cshtml?

**Comment by jrestall** [comment]
I'm no expert on the liquid templating but to me it looks like ShapePagerTag only supports the four defined PagerSlim properties (previous_text , next_text, previous_class, next_class) plus item_classes. So probably a matter of adding in support for the other list shape arguments in this class. 

https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Tags/ShapePagerTag.cs#L17

**Comment by sebastienros** [comment]
Is it still an issue?

**Comment by DannyT** [comment]
I believe it is, I'm off for a week but will take a look and confirm when I get back

**Comment by Skrypt** [comment]
Pagers did not had any style with Themes which is fixed with #5181
We now include the bootstrap 4 pager styles in all our frontend themes.

**Comment by Skrypt** [comment]
I'm looking at this one. This should work as the base shape of a PagerShape is a List from CoreShapes. Lists Tag and ItemTag can be changed.
